### DISCUSSION TITLE
prisma socket file getting deleted - how to fix?

### DISCUSSION THREAD

**Discussion by simbo** [prisma/prisma#4706] [Q&A] [answered]
Hi,

I am using umami, which uses prisma.

As the `/tmp` on my server (uberspace) gets deleted from time to time, the `prisma-*.sock` file can't be found anymore, which lets umami die.

I already reported this issue to the developer of umami [here](https://github.com/mikecao/umami/issues/409) and got informed that this would be more of a prisma problem.

I guess, a temporary file getting removed is a usual thing...

What would be the solution to this problem?

**Comment by janpio** [comment]
This sounds like something we want to investigate. Can you please open an issue and fill the complete issue template so we have all information? Thanks.

**Reply by pantharshit00** [reply]
Yes, @simbo please open an issue so that we can investigate and maybe provide a way of customizing this.

For now, you can use the `@prisma/cli@dev` channel and turn off unix domain sockets connection mode using the following code:

```ts
const prisma = new PrismaClient({
  __internal:{
    useUds: false
  }
} as any) // remove as any if you using js
```


Please note that this only works in the dev channel right now.

**Comment by simbo** [comment, marked as answer]
Issue created: https://github.com/prisma/prisma/issues/4744

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
